presto-raptor/src/test/java/com/facebook/presto/raptor/storage/TestOrcStorageManager.java

```diff
@@ -678,7 +679,6 @@ private static SqlDate sqlDate(int year, int month, int day)
 
     private static SqlTimestamp sqlTimestamp(int year, int month, int day, int hour, int minute, int second)
     {
-        DateTime dateTime = new DateTime(year, month, day, hour, minute, second, 0, UTC);
-        return new SqlTimestamp(dateTime.getMillis(), UTC_KEY);
+        return sqlTimestampOf(year, month, day, hour, minute, second, 0, UTC, UTC_KEY, SESSION);
```

Comment:
See how this `private static SqlTimestamp sqlTimestamp` method is used. Probably a better fix for it would be 
to detach it completely from `SqlTimestamp`, like 

```
long timestampUtc(...) {
    return new DateTime(year, month, day, hour, minute, second, 0, UTC).getMillis()
}
```